src/payment/strategies/stripev3/stripev3-payment-strategy.ts

```diff
@@ -43,7 +43,7 @@ export default class StripeV3PaymentStrategy implements PaymentStrategy {
             throw new MissingDataError(MissingDataErrorType.MissingPaymentMethod);
         }
 
-        return this._stripeScriptLoader.load(paymentMethod.initializationData.stripePublishableKey)
+        return this._stripeScriptLoader.load(paymentMethod.initializationData.stripePublishableKey, paymentMethod.initializationData.stripeConnectedAccount)
```

Comment:
this line seem to have got a bit too long (prefer 120 char max limit per line)

```js
return this._stripeScriptLoader.load(
  paymentMethod.initializationData.stripePublishableKey, 
  paymentMethod.initializationData.stripeConnectedAccount
)
  .then(stripeJs => {
```

Also not required, but it might be good if you can change this line to use async/await
```js
const stripeJs = await this._stripeScriptLoader.load(
  paymentMethod.initializationData.stripePublishableKey, 
  paymentMethod.initializationData.stripeConnectedAccount
);

this._stripeV3Client = stripeJs;
// ...
```